pkg/codegen/dotnet/gen_program.go

```diff
@@ -424,6 +425,10 @@ func (g *generator) usingStatements(program *pcl.Program) programUsings {
 	}
 }
 
+func configObjectTypeName(variableName string) string {
+	return Title(variableName) + "Args"
```

Comment:
Is this just for module/component inputs or are we using this for top level config as well? Not totally sure about calling it "Args" for the top level, but it might do.